src/components/SingleAlbum.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+function SingleAlbum(props) {
+  const { album } = props;
```

Comment:
Acá usaría _object destructuring_ para sacar los _props_, para dejar un código más limpio:
```suggestion
function SingleAlbum({ album }) {
```

Reply:
Oka!! Lo cambiare